<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesystemInfoForm</class>
 <widget class="QWidget" name="FilesystemInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filesystem information</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Temporary directory</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="tmpDirLineEdit">
     <property name="minimumSize">
      <size>
       <width>437</width>
       <height>21</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="tmpDirWarningLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="openTmpButton">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>User data directory</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="usrDataDirLineEdit">
     <property name="minimumSize">
      <size>
       <width>437</width>
       <height>21</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="usrDataDirWarningLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="openUsrButton">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>System data directory</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="sysDataDirLineEdit">
     <property name="minimumSize">
      <size>
       <width>437</width>
       <height>21</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="sysDataDirWarningLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="openSysButton">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
